scripts/dev_scripts/samples_permissions_checker.sh

```
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2022 - 2025, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+#
+# Checks if the files in tests/malware_analyzer/pypi/resources/sourcecode_samples have executable permissions,
+# failing if any do.
+#
+
```

Would it be good to add `set -euo pipefail` in this script?
What we really want to prevent is commands failing silently, leading to an incorrect `SAMPLES_PATH` value, which will make the result of this script wrong.

---

Yes I agree. I'll do that now.